tests/end_to_end/test_vulnerability_detector/conftest.py

```diff
@@ -0,0 +1,46 @@
+"""
+Pytest Configuration for Wazuh Vulnerability Detector End-to-End Testing
+
+This module contains Pytest fixtures and configuration settings for Wazuh end-to-end testing.
+It provides reusable fixtures and setup that can be shared across multiple test modules.
+
+Fixtures:
+    - `host_manager`: Fixture for creating a HostManager instance representing the Wazuh test environment.
+
+Configuration Options:
+    - `--inventory-path`: Path to the inventory file specifying the test environment hosts.
+
+Example:
+    To use the `host_manager` fixture in your test module:
+
+    ```python
+    def test_example(host_manager):
+        # Your test logic using the host_manager fixture
+        pass
+    ```
+"""
+import pytest
+
+from wazuh_testing.tools.system import HostManager
+
+
+@pytest.fixture(scope='session')
+def host_manager(request):
+    """Fixture for creating a HostManager instance.
+
+    This fixture initializes and provides a HostManager instance for testing purposes. The HostManager
+    is set up at the session scope, ensuring it is created only once for the entire test suite.
+
+    Args:
+        request (FixtureRequest): The Pytest fixture request object.
+
+    Returns:
+        HostManager: An instance of the HostManager class representing the Wazuh test environment.
+
+    Raises:
+        ValueError: If the specified inventory path is invalid or not provided.
+    """
+    inventory_path = request.config.getoption('--inventory-path')
+    manager = HostManager(inventory_path)
```

Comment:
```suggestion
    host_manager = HostManager(inventory_path)
```

Could be confused with Wazuh's manager.

Reply:
Done 9aa7fa5a8c25e5d5a80c1063f76eba0051a6d175